pool/pool.go

```diff
@@ -105,19 +105,28 @@ type pool struct {
 
 func newPool(ctx context.Context, options *BuilderOptions) (Pool, error) {
 	clientPacks := make([]*clientPack, len(options.weights))
+	var atLeastOneHealthy bool
 	for i, address := range options.addresses {
 		c, err := options.clientBuilder(client.WithDefaultPrivateKey(options.Key),
 			client.WithURIAddress(address, nil),
 			client.WithDialTimeout(options.NodeConnectionTimeout))
 		if err != nil {
 			return nil, err
 		}
+		var healthy bool
 		st, err := c.CreateSession(ctx, options.SessionExpirationEpoch)
 		if err != nil {
-			return nil, fmt.Errorf("failed to create neofs session token for client %v: %w", address, err)
+			fmt.Printf("failed to create neofs session token for client %v: %s\n", address, err.Error())
```

Comment:
I am not sure printing into stdout is a great behaviour for the library. I would rather add zap.Logger to `BuilderOptions` and use if it is provided.